### DISCUSSION TITLE
Guys, can someone set this up for me?

### DISCUSSION THREAD

**Discussion by cinetici** [itsToggle/plex_debrid#523] [Q&A]
Hi guys. Its my first day using linux-ubuntu and I cant get past many lines provided to achieve this wonderful world of plex-debrid. I would like to offer a paypal beer or a generous coffee to anyone who can set this all up for me with all the servers and trackers and all...

Can someone help me out step by step to correct what im doing wrong or just doing it all for me please? 

plex and real-debrit are up and running just fine! Only interested in 4k content.

**Comment by GSKang94** [comment]
Hey I was able to set it up yesterday. Lemme know if you have any questions.
